lib/openstudio-standards/infiltration/infiltration.rb

```diff
@@ -0,0 +1,183 @@
+    def self.adjust_infiltration_to_prototype_building_conditions(infiltration_rate, initial_pressure: 75.0)
+      alpha = 0.22 # terrain adjustment factor for an urban environment, unitless
+      uh = 4.47 # wind speed, m/s
+      rho = 1.18 # air density, kg/m^3
+      cs = 0.1617 # positive surface pressure coefficient, unitless
+      n = 0.65 # infiltration coefficient, unitless
+
+      # Calculate the typical pressure - same for all building types
+      final_pressure_pa = 0.5 * cs * rho * uh**2
+      adjusted_infiltration_rate = (1.0 + alpha) * infiltration_rate * (final_pressure_pa / initial_pressure)**n
+
+      return adjusted_infiltration_rate
+    end
+
+    # @!endgroup Calculations
+
+    # @!group Surface
+
+    # Determine the component infiltration rate for a surface
+    # Details described in Gowri, Krishnan, Winiarski, David W, and Jarnagin, Ronald E. Infiltration modeling guidelines for commercial building energy analysis. United States: N. p., 2009. Web. doi:10.2172/968203.
+    #
+    # @param surface [OpenStudio::Model::Surface] OpenStudio Surface object
+    # @param type [String] choices are 'baseline' and 'advanced'
+    # @return [Double] component infiltration rate in m^3/s
+    def self.surface_component_infiltration_rate(surface, type: 'baseline')
+      # Define the envelope component infiltration rates
+      component_infil_rates_cfm_per_ft2 = {
+        'baseline' => {
+          'roof' => 0.12,
+          'exterior_wall' => 0.12,
+          'below_grade_wall' => 0.12,
+          'floor_over_unconditioned' => 0.12,
+          'slab_on_grade' => 0.12
+        },
+        'advanced' => {
+          'roof' => 0.04,
+          'exterior_wall' => 0.04,
+          'below_grade_wall' => 0.04,
+          'floor_over_unconditioned' => 0.04,
+          'slab_on_grade' => 0.04
+        }
+      }
+
+      # Skip non-outdoor surfaces
+      boundary_condition = surface.outsideBoundaryCondition
+      unless boundary_condition == 'Outdoors' || boundary_condition == 'Ground'
+        return 0.0
+      end
+
+      # Per area infiltration rate for this surface
+      surface_type = surface.surfaceType
+      infil_rate_cfm_per_ft2 = nil
+      case boundary_condition
+      when 'Outdoors'
+        case surface_type
+        when 'RoofCeiling'
+          infil_rate_cfm_per_ft2 = component_infil_rates_cfm_per_ft2[type]['roof']
+        when 'Wall'
+          infil_rate_cfm_per_ft2 = component_infil_rates_cfm_per_ft2[type]['exterior_wall']
+        when 'Floor'
+          infil_rate_cfm_per_ft2 = component_infil_rates_cfm_per_ft2[type]['floor_over_unconditioned']
+        end
+      when 'Ground'
+        case surface_type
+        when 'Wall'
+          infil_rate_cfm_per_ft2 = component_infil_rates_cfm_per_ft2[type]['below_grade_wall']
+        when 'Floor'
+          infil_rate_cfm_per_ft2 = component_infil_rates_cfm_per_ft2[type]['slab_on_grade']
```

Comment:
Table 1 of https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-18898.pdf says below-grade walls and slab-on-grade floors are 'not used in infiltration calculations'.

Reply:
I'll remove slabs and below grade walls